### PR TITLE
SDK cleanup

### PR DESCRIPTION
**Pull request by FOr3st**
## Changes:
* Types cleanup
* Comments cleanup
* Code cleanup

### CODE REVIEW

**Review by EdNoepel — APPROVED** [review]
Thank you for this tedium.  I was on the fence about periods at the end of comment descriptions, as most of these comments are not complete sentences.  However, they do improve readability and serve as delimiters of ideas, and as such, improve readability.

Much appreciated.

File: src/classes/Bucket.ts [hunk 1]
```diff
@@ -234,7 +234,7 @@ export class Bucket {
   }
 
   /**
-   * checks a lender's LP balance in a bucket
+   * Checks a lender's LP balance in a bucket.
```
Comment (MikeHathaway) [inline src/classes/Bucket.ts:237]: nit: `Retrieves` would be more accurate than `Checks`

File: src/classes/Bucket.ts [hunk 2]
```diff
@@ -273,7 +273,7 @@ export class Bucket {
   };
 
   /**
-   * allows lender to kick a loan based on a LUP calculated as if they withdraw liquidity
+   * Allows lender to kick a loan based on a LUP calculated as if they withdraw liquidity.
```
Comment (MikeHathaway) [inline src/classes/Bucket.ts:276]: nit: `Kick` a loan with a lender's liquidity based on a LUP calculated as if they withdraw liquidity

Comment (FOr3st) [reply src/classes/Bucket.ts]: Adressed in 4dc9c72f655776daa1b9a36f70526e836760235f

File: src/classes/Bucket.ts (hunk 1 above)
Comment (FOr3st) [reply src/classes/Bucket.ts]: Addressed in 4dc9c72f655776daa1b9a36f70526e836760235f

**Review by MikeHathaway — APPROVED** [review]
LGTM! Much cleaner now.

**Pull request merged**